.travis.yml

```diff
@@ -102,10 +102,12 @@ before_install:
   # Use pip 9, so that setuptools can be downgraded.
   # https://github.com/coala/coala/issues/4708
   # Apart from builds with python 2.7 and 3.5, it installs setuptools
-  # twice. So, it is required to uninstall one version manually.
+  # multiple times. So, it is required to uninstall manually.
   - pip uninstall setuptools --yes
   - pip uninstall setuptools --yes || true
   - pip uninstall setuptools --yes || true
+  - pip uninstall setuptools --yes || true
+  - pip uninstall setuptools --yes || true
```

Comment:
This is absurd, do you know why this is required?

Reply:
More than 2 versions of setuptools are getting installed which needs to manually uninstalled.

Reply:
This should not happen, so there's something wrong in the build then